docs/references/what_are_related_usns.md

```diff
@@ -0,0 +1,15 @@
+# What are related USNs
+
+A [USN](what_is_a_usn.md) is composed by different CVEs. If a CVE appears on multiple USNs, we say
+that those USNs are related. On the following image, we can see visual representation of that
+concept:
+
+![Related USN example](../images/usn-related.png)
```

Comment:
I really like the diagram, I think it shows the hierarchy really clearly, but would it be possible to change the colours used? A significant chunk of humans are red/green colourblind so using those colours together is usually better avoided.

Reply:
Good point, I can change that